packages/desktop/features/wallet.features.ts

```diff
@@ -18,12 +18,12 @@ const walletFeatures: IWalletFeatures = {
         },
     },
     activityHistory: {
```

Comment:
why should the activity history be disabled? :thinking:

Reply:
I confused `activityHistory` with `addressHistory` :trollface: